common/dynamicconfig/constants.go

```diff
@@ -2133,6 +2133,16 @@ that task will be sent to DLQ.`,
 		10,
 		`Maximum number of resend events batch for a single replication request`,
 	)
+	WorkflowIdReuseMinimalInterval = NewNamespaceDurationSetting(
+		"history.workflowIdReuseMinimalInterval",
```

Comment:
note I renamed this from `system` to `history` to be consistent with other flags we have in history service.